app/src/components/SearchInput.tsx

```diff
@@ -0,0 +1,63 @@
+import { InputGroup } from "@/components/ui/input-group";
+import { Icon, IconButton, Input } from "@chakra-ui/react";
+import { LuSearch } from "react-icons/lu";
+import { MdClose } from "react-icons/md";
+import React from "react";
+
+const SearchInput = ({
+  searchTerm,
+  setSearchTerm,
+  className,
+  placeholder,
+}: {
+  searchTerm: string;
+  setSearchTerm: (searchTerm: string) => void;
+  className?: string;
+  placeholder?: string;
+}) => {
+  return (
+    <InputGroup
+      color=""
+      w="full"
+      rounded={32}
+      backgroundColor="background.overlay"
+      className={className}
+      startElement={
+        <Icon
+          as={LuSearch}
+          className="opacity-50"
+          color={"#414249"}
+          boxSize={4}
+        />
+      }
+      endElement={
+        searchTerm ? (
+          <IconButton
+            onClick={() => setSearchTerm("")}
+            aria-label="search"
+            colorScheme="interactive.secondary"
+            variant="ghost"
+          >
+            <Icon
+              as={MdClose}
+              className="opacity-50"
+              boxSize={4}
+              color={"#414249"}
```

Comment:
Can you use a color name from `app-theme.ts` instead of the hardcoded color?
They should be supported in Chakra's `Icon` component.